core/src/main/java/io/smallrye/openapi/runtime/scanner/dataobject/AnnotationTargetProcessor.java

```diff
@@ -189,10 +190,41 @@ Schema processField() {
             fieldSchema = MergeUtil.mergeObjects(typeSchema, fieldSchema);
         }
 
+        processFieldAnnotations(fieldSchema, typeResolver);
+
```

Comment:
@abatalev - sorry, one more. I think this method call should occur before the if/else starting on line 177. The reason is because the XML changes should be applied prior to the potential schema composition that occurs in that condition. 

The other thing I would mention is that `processFieldAnnotations` will not find method annotations if an unannotated field is present in the class (likewise for the writer method, if a reader method is present).

Reply:
Done